src/components/list/ListDisplay.jsx

```diff
@@ -0,0 +1,44 @@
+
+import React from 'react';
+import { useBreeds } from '../../hooks/breeds';
+import ListItem from './ListItem';
+
+const ListDisplay = () => {
+
+    const { loading, breeds, page, setPage } = useBreeds();
+
+    return (
+
+        <>
+
+            {loading ?
+                <div>Loading...</div>
+
+                : <div data-testid="breeds"> <ul >
+                    {breeds.map(breed => (
+                        <li key={breed.id} style={{ margin: '50px' }}>
+                            < ListItem {...breed} />
+                        </li>
+                    ))};
+        </ul>
+                </div>}
+
+            <div>{page === 0 ?
```

Comment:
alternatively. You can use disabled property for the button. So on like 29 you could do 
``` 
 <button disabled={page === 1} onClick={() => setPage(page - 1)}>previous page</button>
```
that will keep the button on the page but be grayed out. Both are fine and great solution.